src/main/java/frc/robot/subsystems/climb/ClimbIOVictorSPX.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems.climb;
+
+import com.ctre.phoenix.motorcontrol.VictorSPXControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+import frc.robot.Constants;
+
+public class ClimbIOVictorSPX implements ClimbIO
+{
+    public final VictorSPX           _climbVictorSPXLeft;
+    public final VictorSPX           _climbVictorSPXRight;
+    public final AnalogPotentiometer _potentiometerLeft;
+    public final AnalogPotentiometer _potentiometerRight;
+    public final double              _leftOffset  = Constants.Climb.LEFT_ZERO_OFFSET;
+    public final double              _rightOffset = Constants.Climb.RIGHT_ZERO_OFFSET;
+
+    public ClimbIOVictorSPX()
+    {
+        _climbVictorSPXLeft  = new VictorSPX(Constants.CAN.CLIMB_LEFT);
+        _climbVictorSPXRight = new VictorSPX(Constants.CAN.CLIMB_RIGHT);
+
+        _potentiometerLeft  = new AnalogPotentiometer(Constants.AIO.CLIMB_LEFT_SENSOR);
+        _potentiometerRight = new AnalogPotentiometer(Constants.AIO.CLIMB_RIGHT_SENSOR);
```

Comment:
The `AnalogPotentiometer` constructor can take another parameter that represents the scaling factor. We should create a constant for the scale and pass it in here. That way, we don't have to worry about applying the scale ourselves:

```suggestion
        _potentiometerLeft  = new AnalogPotentiometer(Constants.AIO.CLIMB_LEFT_SENSOR, Constants.Climb.SENSOR_SCALE);
        _potentiometerRight = new AnalogPotentiometer(Constants.AIO.CLIMB_RIGHT_SENSOR, Constants.Climb.SENSOR_SCALE);
```